### ISSUE TITLE
Define a minimum access level to parse the documentation

### ISSUE THREAD

**Issue by minuscorp** [eneko/SourceDocs#27]
## Steps to Reproduce

`sourcedocs generate --output-folder Docs -- -workspace Ellu.xcworkspace -scheme Ellu CODE_SIGNING_ALLOWED="NO"`

### Expected Outcome

Documentation for a selected minimum access level see [this](https://github.com/realm/jazzy#controlling-what-is-documented) for reference

## Environment

- **Operating System:** macOS 10.15.0
- **Xcode Version:** 11.1
- **SwiftDocs Version/SHA:** 0.5.1
- **Swift version:** 5.1

I think I could provide a fast PR to address this feature

**Comment by minuscorp** [comment]
To add more information about a probably related issue, running:
`sourcedocs generate --output-folder Docs -- -workspace Ellu.xcworkspace -scheme Ellu CODE_SIGNING_ALLOWED=NO`
Does not recognize none files in the project, while:
`sourcekitten doc -- -workspace Ellu.xcworkspace -scheme Ellu > swiftDoc.json`
Produces:
```
Running xcodebuild
Parsing AppDelegate.swift (1/21)
Parsing Assembler.swift (2/21)
Parsing AssemblerType+Test.swift (3/21)
Parsing AssemblerType.swift (4/21)
Parsing AutoCopy.generated.swift (5/21)
Parsing AutoEquatable.generated.swift (6/21)
Parsing AutoEquatable.swift (7/21)
Parsing AutoEquatableAction.generated.swift (8/21)
Parsing AutoHashable.generated.swift (9/21)
Parsing AutoHashable.swift (10/21)
Parsing StatefulController.swift (11/21)
Parsing StatefulViewController.swift (12/21)
Parsing ActivityIndicator.swift (13/21)
Parsing ObservableType+Handle.swift (14/21)
Parsing ErrorResolver.swift (15/21)
Parsing NavigatorType.swift (16/21)
Parsing ViewModelBindable.swift (17/21)
Parsing ErrorHandler.swift (18/21)
Parsing ErrorHandlerType.swift (19/21)
Parsing ViewController.swift (20/21)
Parsing ViewModelType.swift (21/21)
```
So or we implement a method to let *SourceDocs* to parse a given *SourceKitten* `json` file to generate the docs or fix the issue with the projects with SPM

**Comment by minuscorp** [comment]
fixed in #28
